bin/auto-cpufreq

```diff
@@ -212,14 +215,41 @@ def main(config, daemon, debug, install, remove, live, log, monitor, stats, vers
                         auto_cpufreq_stats_file.close()
 
                     auto_cpufreq_stats_path.unlink()
-                # ToDo:
+                # ToDo: 
+                # {the following snippet also used in --update, update it there too(if required)}
                 # * undo bluetooth boot disable
                 gnome_power_rm_reminder_snap()
                 remove_complete_msg()
             else:
                 root_check()
                 remove_daemon()
                 remove_complete_msg()
+        elif update:
+            root_check()
+            if os.getenv("PKG_MARKER") == "SNAP":
+                print("Detected auto-cpufreq was installed using snap")
+                # subprocess.run(["snap", "refresh", "auto-cpufreq"])
```

Comment:
Let's remove comment on 231

Reply:
can i add a to-do here? and maybe create an issue, maybe some genius can find a way out

Reply:
Please do :)